neuralmonkey/readers/plain_text_reader.py

```diff
@@ -19,6 +23,38 @@ def reader(files: List[str]) -> Iterable[List[str]]:
     return reader
 
 
+def column_separated_reader(column: int, delimiter: str = "\t",
```

Comment:
to bohuzel nelze lehce udelat z duvodu, ze to vyuziva get_plain_text_reader ktery muze dostat seznam souboru a tomu CSV nerekne kdy zacal cist z dalsiho.

Navic standardne (ac CSV neni standardizovane) je header pouze optional a spise mate implementace (napriklad default pythoni vubec s hlavickou nepocita a musi se o to postarat sam programator, co knihovnu pouziva).